LiskHQ/lisk-hub
# Refactor route definition - Closes #499

### What was the problem?
Some routes definitions were in the string
### How did I fix it?
Created constants for route
### How to test it?
Search for `main` and `explorer` to check if route definition comes from one file
### Review checklist
- The PR solves #499 
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
